documentation/how-to/scaling.md

```diff
@@ -0,0 +1,357 @@
+            }
+          ]
+        },
+        {
+          "role": "worker",
+          "memberCount": 1,
+          "members": [
+            {
+              "identity": "10.1.232.42:7239"
+            }
+          ]
+        }
+      ]
+    },
+    "clusterId": "514dd854-eb26-4e93-9f3c-1355cb8aa99a",
+    "clusterName": "active",
+    "historyShardCount": 4,
+    "persistenceStore": "postgres",
+    "visibilityStore": "postgres",
+    "failoverVersionIncrement": "10",
+    "initialFailoverVersion": "1"
+  }
+result: command succeeded
+```
+
+## Adding Replicas
+
+To add more replicas you can use the juju scale-application functionality i.e.
+
+```
+juju scale-application temporal-k8s -n <num_of_replicas_required_replicas>
+```
+
+To scale all four services to two units each, you can run the following
+commands:
+
+```
+juju scale-application temporal-k8s -n 2
+juju scale-application temporal-k8s-history -n 2
+juju scale-application temporal-k8s-matching -n 2
+juju scale-application temporal-k8s-worker -n 2
+```
+
+You can then run `juju status --watch 1s` to watch the status of your
+applications. It may take a few minutes to see the following output where all
+nodes are showing `Workload=Active` and `Agent=idle`:
+
+```
+Model     Controller           Cloud/Region        Version  SLA          Timestamp
+temporal  temporal-controller  microk8s/localhost  3.1.0    unsupported  15:18:19+03:00
+
+App                    Version  Status  Scale  Charm               Channel    Rev  Address         Exposed  Message
+postgresql-k8s         14.7     active      1  postgresql-k8s      14/stable   73  10.152.183.171  no       Primary
+temporal-admin-k8s              active      1  temporal-admin-k8s  edge         4  10.152.183.35   no
+temporal-k8s                    active      2  temporal-k8s                     0  10.152.183.95   no
+temporal-k8s-history            active      2  temporal-k8s                     1  10.152.183.52   no
+temporal-k8s-matching           active      2  temporal-k8s                     2  10.152.183.122  no
+temporal-k8s-worker             active      2  temporal-k8s                     3  10.152.183.134  no
+
+Unit                      Workload  Agent  Address      Ports  Message
+postgresql-k8s/0*         active    idle   10.1.232.60         Primary
+temporal-admin-k8s/0*     active    idle   10.1.232.20
+temporal-k8s-history/0*   active    idle   10.1.232.6
+temporal-k8s-history/1    active    idle   10.1.232.51
+temporal-k8s-matching/0*  active    idle   10.1.232.61
+temporal-k8s-matching/1   active    idle   10.1.232.38
+temporal-k8s-worker/0*    active    idle   10.1.232.42
+temporal-k8s-worker/1     active    idle   10.1.232.23
+temporal-k8s/0*           active    idle   10.1.232.26
+temporal-k8s/1            active    idle   10.1.232.25
+```
+
+To confirm the four scaled services can reach other, you can run
+`juju run temporal-admin-k8s/0 tctl args="adm cl d"`, you should see the
```

Comment:
Is this bit plus the output not just a repeat of the above? I.e. can we just say refer to above or something similar.

Reply:
I could remove it, but I've included a second copy because I thought it would be useful to see how the output would look like when we have replicas (i.e. the way each service has two reachable units).

Reply:
Gotcha, that's fine in that case.